FizzBuzz/FizzBuzz.cs

```diff
@@ -7,26 +7,38 @@ class Program
         //Main Code
         static void Main(string[] args)
         {
-            Console.WriteLine("Please choose enter 1 or 2 for the loop method:" +
+Console.WriteLine("Please choose enter 1 or 2 for the loop method:" +
             "(1) 'For Loop'" +
             "(2) 'While Loop'");
             int Loop_Type = 0;
 
             //Read Loop Type Input
-            Loop_Type = (Convert.ToInt32(Console.ReadLine()));
-
+            try
+            {
+                Loop_Type = (Convert.ToInt32(Console.ReadLine()));
+                if (Test1(Loop_Type))
+                {
+                    Loop(Loop_Type);
+                }
+                else Console.WriteLine("Please only enter the value 1 or 2!");
+            }
+            catch
+            {
+                //If the value entered in by Reader is not 1 or 2, run this response instead.
+                Console.WriteLine("Please only enter the value 1 or 2!");
+            }
             //Run the Loop Type
-            Loop(Loop_Type);
-
+            
             //Code here so program does not end automatically
             Console.ReadLine();
         }
-        //Loop Codes
+        //Loop Codes go here
         public static void Loop(int Loop_Type)
         {
             //Loop Code Type 1
             if (Loop_Type == 1)
             {
+                Console.WriteLine("Running the For Loop:");
                 //For the int value i, while the value is less than 101, run the code below. Then increment by 1
                 for (int i = 0; i < 101; i++)
```

Comment:
The output of Loop_Type == 1 is not the same as Loop_Type == 2. Did you intend to do that?